src/App.tsx

```diff
@@ -60,6 +61,7 @@ function App() {
         {path:'facilities/facilities-form',element:<FacilitiesForm/>},
         {path:'rooms',element:<RoomsList/>},
         {path:'rooms/rooms-form',element:<RoomsForm/>},
+        {path:'rooms/updateRoom/:roomId',element:<UpdateRoom/>},
```

Comment:
we need to be consistent when naming routes, following other routes. We should update the path to `rooms/update-room/:room-id`